bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -73,9 +73,9 @@ metadata:
           "metadata": {
             "labels": {
               "app.kubernetes.io/created-by": "opendatahub-operator",
-              "app.kubernetes.io/instance": "default",
+              "app.kubernetes.io/instance": "default-feature",
```

Comment:
this probably was my fault, i pointed out to update them in the previous PR but not done it properly in the source config/samples/featurectracker_v1_features.yaml, thats why you got it reverted after generate manifests.
could you update that one instead ?

Reply:
I just run `make bundle` and this change popped up. I modified `config/samples/featurectracker_v1_features.yaml` to match the bundle's values. Please LMK if this is what you meant.